beep/utils/events.py

```diff
@@ -354,6 +355,128 @@ def put_generate_event(self, output_data, generate_status):
         return response
 
 
+class WorkflowOutputs:
+    """
+    Attributes:
+        service (str): default: Which service is instantiating the class so that
+        we know who is sending events. Defaults to 'Testing'.
+        mode (str): Mode for events, test will only put test message. Defaults to 'run'.
+    """
```

Comment:
Looks the docstring is carried over from class KinesisEvents and needs to be updated?

Reply:
👍